css/style.css

```diff
@@ -1,5 +1,63 @@
 /* General page styles */
 
-
 /* We should use a mobile first approach having out breack point at 768px */
 
+.button {
+	font-family: roboto, sans-serif;
+	font-weight: 500;
+	border-radius: 2px;
+	overflow: hidden;
```

Comment:
Why did you set this property?

Reply:
That was to avoid bad behavior of the button when the user attempt content that overflows the designed width of it.